schedule/containers/engines_and_tools.yaml

```diff
@@ -6,24 +6,45 @@ description:    >
 conditional_schedule:
   boot:
     ARCH:
-      's390x':
+      s390x:
         - installation/bootloader_start
   validate_btrfs:
     ARCH:
       x86_64:
         - containers/validate_btrfs
+  runc:
+    ARCH:
+      x86_64:
+        - containers/docker_runc
+      s390x:
+        - containers/docker_runc
+  supported:
```

Comment:
why not something that mentions podman (ex: podman_supported) or just podman?

Reply:
cause it's also for buildah package, `buildah_podman`?

Reply:
i guess we can leave it as it is. We can always change it in another time